tools/swift-inspect/Sources/swift-inspect/RemoteMirrorExtensions.swift

```diff
@@ -82,4 +96,13 @@ extension SwiftReflectionContextRef {
     }
     return allocations
   }
+
+  var allocationBacktraces: [swift_reflection_ptr_t: Backtrace] {
+    var backtraces: [swift_reflection_ptr_t: Backtrace] = [:]
```

Comment:
Is there supposed to be a space after `:` in the hash table type?  I don't really know about the proper style here, so this is more a question for my own learning.

Reply:
I'm not really the right person to ask, but `swift-format` seems to think this is right. (It also does a bunch of silly stuff, so it's not entirely trustworthy....)

Reply:
Haha, okay.